.travis.yml

```diff
@@ -21,55 +21,46 @@ env:
     - RUST_NIGHTLY_VERSION=nightly-2018-06-29
     - RUST_CLIPPY_VERSION=0.0.211
     - EJB_RUST_BUILD_DIR="$TRAVIS_BUILD_DIR/exonum-java-binding-core/rust/"
+    - SODIUM_BUILD=1
```

Comment:
Why it's re-added? @dmitry-timofeev 's comments are marked as resolved, but they are definitely not